packages/core/content-releases/server/src/services/scheduling.ts

```diff
@@ -0,0 +1,53 @@
+import { scheduleJob, Job } from 'node-schedule';
+import { LoadedStrapi } from '@strapi/types';
+
+import { errors } from '@strapi/utils';
+import { Release } from '../../../shared/contracts/releases';
+import { getService } from '../utils';
+import { RELEASE_MODEL_UID } from '../constants';
+
+const createSchedulingService = ({ strapi }: { strapi: LoadedStrapi }) => {
+  const scheduleJobs = new Map<Release['id'], Job>();
```

Comment:
another benefit of using "strapi.cron" would be to not need this map in a content-releases service.

Atm, if using this service, everytime the service is instantiated wouldn't you be creating a new map and lose the old schedulings ?

Reply:
I don't quite recall how services are loaded 🤔 if they were loaded only once on startup or every time you request a service, so maybe I am wrong

Reply:
They are loaded only once on startup, so that part is okay. My plan was on bootstrap load the `scheduleJobs` with the releases with schedule date from database (this is to handle a reset on the server). And cancel all of them on destroy.

But I would take a look at `strapi.cron` I agree is better idea use the same for all the repo

Reply:
Oh, if they are loaded only once on startup that would be ok :) 

Also, I believe strapi.cron removes all crons when resetting the server, so that should be handled automatically